Sources/GoTrue/Types.swift

```diff
@@ -384,19 +385,25 @@ public struct OTPParams: Codable, Hashable, Sendable {
   public var createUser: Bool
   public var data: [String: AnyJSON]?
   public var gotrueMetaSecurity: GoTrueMetaSecurity?
+  public var codeChallenge: String?
+  public var codeChallengeMethod: String?
```

Comment:
Would be nice to have a `enum` representing the codeChallgendMethod, even we if we have single method.

```swift
public enum PKCECodeChallengeMethod: String {
    case sha256 = "S256"
}
```